tools/debug/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "debug_tools"
+version = "0.0.1"
+edition = "2021"
+description = "Debugging tools for Emergence"
+publish = false
```

Comment:
hmm, do we need `publish = false`?

Reply:
Not technically, though I figured since it was a project specific crate and in an early state that it may be safe to prevent accidental publishing.

Reply:
@wallamide makes sense!